src/App.tsx

```diff
@@ -21,14 +33,15 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {(todos?.length > 0)
+                ? <TodoList todos={todos} />
+                : <Loader />}
```

Comment:
if there will be no data due to an error, the loading state will persist till the error is solved, please provide some kind of loading state and base your ternary upon that